src/Symfony/Component/Yaml/Parser.php

```diff
@@ -51,6 +52,35 @@ public function __construct()
     }
 
     /**
+     * Parses a YAML file into a PHP value.
+     *
+     * @param string $filename A string containing YAML
```

Comment:
Please update this:

```
@param string $filename A string containing YAML
```

to something like this:

```
@param string $filename A relative or absolute path to the YAML file
```

Same for the `Yaml:: parseFile()` method below.

By the way ... should we replace `$filename` by `$filepath` everywhere in the code of this PR?